pyflakes/messages.py

```diff
@@ -233,6 +233,13 @@ class TooManyExpressionsInStarredAssignment(Message):
     message = 'too many expressions in star-unpacking assignment'
 
 
+class IfTuple(Message):
+    """
+    Conditional test is a tuple, which are always True.
```

Comment:
```suggestion
    Conditional test is a non-empty tuple literal, which are always True.
```

(The existing docstring for `AssertTuple` is inaccurate in the same way, too.)

Reply:
Nice :+1:

Reply:
I've separately put up https://github.com/PyCQA/pyflakes/pull/522 for `AssertTuple`